src/v6_indexer/processor.ts

```diff
@@ -336,12 +338,36 @@ async function handleLaunchCompletedEvent(event: LaunchCompletedEvent, signature
         }
       }
 
-      await trx.update(schema.v0_6_launches).set({ 
+      // Fetch the launch account to get additional fields
+      let launchUpdateData: Partial<typeof schema.v0_6_launches.$inferInsert> = {
         totalCommittedAmount: BigInt(event.totalCommitted.toString()),
         state: launchState,
         seqNum: BigInt(event.common.launchSeqNum.toString()),
         daoAddr: launchState === V06LaunchState.Complete ? event.dao?.toString() : null,
-      }).where(eq(schema.v0_6_launches.launchAddr, event.launch.toString()));
+      };
+
+      // If launch is complete, fetch additional data from the launch account
+      if (launchState === V06LaunchState.Complete) {
+        try {
+          const launchAccount = await launchpadClient.getLaunch(event.launch);
+          if (launchAccount) {
+            launchUpdateData = {
+              ...launchUpdateData,
+              finalRaiseAmount: launchAccount.finalRaiseAmount ? BigInt(launchAccount.finalRaiseAmount.toString()) : null,
```

Comment:
I'm not a fan of putting bad data into the DB, this is our chance to let ourselves know something is borked vs just everything is fine.